ui/src/submissions/jobs/components/JobForm.jsx

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field, Form } from 'formik';
+import { Col, Row } from 'antd';
+
+import TextField from '../../common/components/TextField';
+import SelectField from '../../common/components/SelectField';
+import ArrayOf from '../../common/components/ArrayOf';
+import SuggesterField from '../../common/components/SuggesterField';
+import SubmitButton from '../../common/components/SubmitButton';
+import { regionOptions } from '../schemas/constants';
+import {
+  arxivCategoryOptions,
+  rankOptions,
+} from '../../common/schemas/constants';
+import DateField from '../../common/components/DateField';
+import RichTextField from '../../common/components/RichTextField';
+import StatusFieldContainer from '../containers/StatusFieldContainer';
+import { isValidDeadlineDate } from '../schemas/job';
+
+class JobForm extends Component {
+  static isInvalidDeadlineDate(date) {
+    return !isValidDeadlineDate(date);
+  }
+
+  // TODO: move them somewhere common to share with `AuthorForm`
+  static getSuggestionSourceLegacyICN(suggestion) {
+    return suggestion._source.legacy_ICN;
+  }
+
+  static getSuggestionSourceLegacyName(suggestion) {
+    return suggestion._source.legacy_name;
+  }
+
+  render() {
+    const { values, isSubmitting, isValid, isValidating } = this.props;
+    return (
+      <Form className="bg-white pa3">
+        <Field
+          name="status"
+          label="* Status"
+          component={StatusFieldContainer}
+        />
+        <Field name="title" label="* Title" component={TextField} />
+        <Field
+          name="external_job_identifier"
+          label="Job ID"
+          placeholder="Used to reference this job opening, e.g. 07845"
+          component={TextField}
+        />
+        <ArrayOf
+          values={values}
+          name="institutions"
+          label="* Institutions"
+          emptyItem={{}}
+          renderItem={itemName => (
+            <Field
+              onlyChild
+              name={`${itemName}.value`}
+              recordFieldPath={`${itemName}.record`}
+              placeholder="Institution, type for suggestions"
+              pidType="institutions"
+              suggesterName="affiliation"
+              extractItemCompletionValue={JobForm.getSuggestionSourceLegacyICN}
+              component={SuggesterField}
+            />
+          )}
+        />
+        <Field
+          name="regions"
+          label="* Regions"
+          mode="multiple"
+          options={regionOptions}
+          component={SelectField}
+        />
+        <Field
+          name="field_of_interest"
+          label="* Field of Research"
```

Comment:
``field of interest``

Reply:
Apparently this is the label we want :/

Reply:
why we didn't added in the schema from the beginning? anyway :/

Reply:
in the schema it's called `arxiv_categories`. @pazembrz insisted we put both `add_arxiv_category` and `add_field_of_interest` to the builder, and now we use a different display name.

Reply:
Since we can't map form fields to schema fields for `authors` and `literature`, I decided that fields should be consistent with labels at least. So the idea was to do the same thing for job form (even though we could more or less use fields names from schema since it's simple for this case), and label for this field was `Field of Interest` in the requirements which then changed.